src/app/modules/notifications/org-structure-notification.service.ts

```
@@ -389,25 +691,39 @@ export class OrgStructureNotificationService {
+        } else if (changeDetails.oldParent && !changeDetails.newParent && changeDetails.oldParent) {
+          details.push(`Department updated from "${changeDetails.oldParent}" to root level`);
+        } else if (!changeDetails.oldParent && changeDetails.newParent) {
+          // When moving from root level to a parent department
+          details.push(`Department moved from root level to "${changeDetails.newParent}"`);
+        } else {
           details.push('Department information has been updated');
         }
         break;
+      case 'department_transfer':
+        // For transfers, show simple message with department name and new parent
+        const departmentName = notification.department?.name || 'Department';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)